Library/Styles/SettingsStyles.swift

```diff
@@ -85,15 +103,3 @@ public func settingsAttributedPlaceholder(_ string: String) -> NSAttributedStrin
     attributes: [NSAttributedString.Key.foregroundColor: UIColor.ksr_text_dark_grey_400]
   )
 }
-
-public func settingsHeaderFooterLabelBaseStyle(_ label: UILabel) -> UILabel {
```

Comment:
Moving these 2 will probably cause some merge conflicts on the `Create password` branch but now that we know about it we should be able to resolve it just fine!